### PR TITLE
Enables table elements in the blog.

### PR DESCRIPTION
**Pull request by michaeldupuisjr**

### CODE REVIEW

**Comment by michaeldupuisjr** [comment]
Hey @acacheung. This was my quick and dirty attempt at getting tables into the blog.

File: config.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def has_illustration?
 end
 
 set :markdown_engine, :redcarpet
-set :markdown, :layout_engine => :erb, :fenced_code_blocks => true, :lax_html_blocks => true, :renderer => ::Highlighter::HighlightedHTML.new
+set :markdown, :tables => true, :layout_engine => :erb, :fenced_code_blocks => true, :lax_html_blocks => true, :renderer => ::Highlighter::HighlightedHTML.new
```
Comment (bcardarella) [inline config.rb]: the markdown-style tables are OK, but you should be able to also just use HTML:

```markdown
<table>
  <thead>
    <tr><th>First Name</th><th>Last Name</th><th>Age</th></tr>
  </thead>
  <tbody>
   <tr><td>Brian</td><td>Cardarella</td><td>35</td></tr>
   <tr><td>Brian</td><td>Cardarella</td><td>35</td></tr>
  </tbody>
</table>
```

**Comment by michaeldupuisjr** [comment]
@bcardarella Want me to pull the markdown tables out or is this branch good to merge with Amanda's changes?

**Comment by bcardarella** [comment]
@michaeldupuisjr up to you, just thought you should be aware of the alternative

**Pull request merged**